test/integration/narrow/routes/farmer-apply/confirmation.test.js

```diff
@@ -4,18 +4,34 @@ const getCrumbs = require('../../../../utils/get-crumbs')
 
 const auth = { credentials: { reference: '1111', sbi: '111111111' }, strategy: 'cookie' }
 
-jest.mock('../../../../../app/lib/send-email', () => () => {
+jest.mock('../../../../../app/lib/send-email', () => {
```

Comment:
This mock can be removed as there is no email sending in the module now.

Reply:
Done